homework-submission/hw-week2/src/App.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import './App.css';
+
+const dynamicList = [
+  {
+    id: 1,
+    description: 'Get out of bed',
+    deadline: '2017-09-11',
+    done: true,
+  },
+  {
+    id: 2,
+    description: 'Brush teeth',
+    deadline: '2017-09-10',
+    done: false,
+  },
+  {
+    id: 3,
+    description: 'Eat breakfast',
+    deadline: '2017-09-09',
+    done: false,
+  },
+];
+
+const Button = ({ text, clickHandler }) => {
+  return <button onClick={clickHandler}>{text}</button>;
+};
+
+const ItemField = ({ formHandler }) => {
+  return (
+    <form onSubmit={formHandler}>
+      <input type="text" name="itemField" />
+      <button type="submit">Add Todo</button>
+    </form>
+  );
+};
+
+const Todo = ({ itemName, items }) => {
+  return (
+    <div className="container">
+      <h1 className="header">{itemName}</h1>
+      <div className="text-area">{items}</div>
+    </div>
+  );
+};
+
+const Item = ({ description, deadline, removeHandler }) => {
+  return (
+    <div className="items">
+      <div className="item">
+        <span id="description">{description}</span>
+        <span id="deadline">{deadline}</span>
+      </div>
+      <Button clickHandler={removeHandler} text="remove" />
+    </div>
+  );
+};
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todo: dynamicList,
+    };
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    if (!e.target.itemField.value) {
+      alert('Text field is empty, please enter todo');
+    } else {
+      const id = this.state.todo.length;
+      const object = {
```

Comment:
It's true that this is an `object`, but actually it's a `todo` right? Try to name your variables so that it is very clear what value they are holding.